libs/s25main/ingameWindows/iwSave.cpp

```diff
@@ -27,18 +27,19 @@
 #include "controls/ctrlTable.h"
 #include "desktops/dskLobby.h"
 #include "files.h"
+#include "helpers/make_array.h"
 #include "helpers/toString.h"
 #include "iwPleaseWait.h"
 #include "network/GameClient.h"
+#include "gameData/GameConsts.h"
 #include "gameData/const_gui_ids.h"
 #include "liblobby/LobbyClient.h"
 #include "s25util/Log.h"
 #include <utility>
 
-const unsigned NUM_AUTO_SAVE_INTERVALS = 7;
-
-const std::array<unsigned, NUM_AUTO_SAVE_INTERVALS> AUTO_SAVE_INTERVALS = {500,   1000,   5000, 10000,
-                                                                           50000, 100000, 1 /*  */};
+// autosave intervals in minutes
+const auto AUTO_SAVE_INTERVALS = helpers::make_array(1u, 5u, 10u, 15u, 30u, 60u, 90u);
```

Comment:
Could you use chrono units here? Makes it easier to extend.

```suggestion
using namespace std::chrono_literals;
const auto AUTO_SAVE_INTERVALS = helpers::make_array(1min, 5min, 10min, 15min, 30min, 60min, 90min);
```

Reply:
I wasn't aware of chrono literals. Useful :)